src/release-notify-action.js

```diff
@@ -32,6 +34,23 @@ async function runAction(token, staleDays, commitMessageLines) {
     staleDays
   )
 
+  const closedNotifyIssue = await getLastClosedNotifyIssue(
+    token,
+    latestRelease.published_at,
+    staleDays
+  )
+
+  if (closedNotifyIssue) {
```

Comment:
I still think the logic is not quite right here. The thing which doesn't look quite right here is that if there is a closed issue, then we create a new one. But that's not quite right, because we're assuming that the existence of a closed issue implies that there are unreleased commits, which can be a fair assumption but not a completely correct one either. I'll try to clarify the requirements better in the issue then.

Reply:
Only if there is a issue from release-notify-bot closed as unplanned after the release date. Ive tested it and basically it only creates a snooze issue if you closed the original as unplanned

Reply:
Marco this logic still doesn't work anyway. Another example: you closed the issue as unplanned (let's say you have 7 stale-days configured). The next day the action runs and this new method `getLastClosedNotifyIssue` will return undefined, because it found an issue closed as unplanned but the date it was closed is less than 7 days ago. Hence this new logic will not be executed and it will continue with the normal execution, which will create a new issue, thereby keeping exactly the same behavior as before. Instead, we want to MUTE this action if there is a closed issue.

Overall, I don't think you can reason about this issue until you change stale-days to something that accepts a time such as in the format of https://www.npmjs.com/package/ms, which allows you to test with shorter timeframes.

This PR doesn't work, I recommend that you close it and start from scratch. By keeping this code around, you're preventing yourself from being able to understand the issue and why this code doesn't work.